jina/types/score/__init__.py

```diff
@@ -37,18 +37,19 @@ class NamedScore(ProtoTypeMixin):
              score_proto.value = 10.0
              score = NamedScore(score_proto)
 
+    :param score: The score to construct from, depending on the ``copy``,
+        it builds a view or a copy from it.
+    :type score: Optional[jina_pb2.NamedScoreProto]
+    :param copy: When ``score`` is given as a :class:`NamedScoreProto` object, build a
+        view (i.e. weak reference) from it or a deep copy from it.
+    :type copy: bool
+    :param kwargs: Other parameters to be set
+
      """
 
     def __init__(self, score: Optional[jina_pb2.NamedScoreProto] = None,
                  copy: bool = False, **kwargs):
-        """
-
-        :param score: the score to construct from, depending on the ``copy``,
-                it builds a view or a copy from it.
-        :param copy: when ``score`` is given as a :class:`NamedScoreProto` object, build a
-                view (i.e. weak reference) from it or a deep copy from it.
-        :param kwargs: other parameters to be set
-        """
+        """Set constructor."""
```

Comment:
why set?

Reply:
It has to be an imperative sentence. I don't know what else could be better wording, define?